loc/index.js

```diff
@@ -30,8 +30,10 @@ const init = async lang => {
   if (lang) {
     strings.setLanguage(lang);
     let localeForDayJSAvailable = true;
+    let dayJsLangName = undefined;
```

Comment:
33 | maybe let dayJsLangName = lang;

and 
66 | dayjs.locale(dayJsLangName);

Reply:
done